docs/tutorials.rst

```diff
@@ -40,12 +43,21 @@ us if you are still confused).
 .. toctree::
    :maxdepth: 1
 
-   tutorials/RV_MCMC_Tutorial.ipynb
-   tutorials/Multiplanet_Tutorial.ipynb
    tutorials/Modifying_Priors.ipynb
-   tutorials/Using_nonOrbitize_Posteriors_as_Priors.ipynb
    tutorials/Plotting_tutorial.ipynb
    tutorials/MCMC_vs_OFTI.ipynb
    tutorials/Modifying_MCMC_initial_positions.ipynb
 
+**Feature Tutorials:**   
+
+.. toctree::
+   :maxdepth: 1
```

Comment:
I think you can add labels to the toctrees btw if it helps with separating them: `    :caption: Setup`

Reply:
Thanks! The reason I separated the tutorials into three subsets is because we now have categories of "tutorials that teach you to use the code in the default use case," "tutorials that teach you more about the underlying API," and "tutorials that teach you to use specific new features." Do you think those descriptors would be helpful to add as captions?